android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<network-security-config>
-    <base-config cleartextTrafficPermitted="true">
+<network-security-config xmlns:tools="http://schemas.android.com/tools">
+    <base-config cleartextTrafficPermitted="false">
         <trust-anchors>
             <certificates src="system" />
-            <certificates src="user" />
+            <certificates src="user"
+                tools:ignore="AcceptsUserCertificates" />
```

Comment:
@grahamsmith Why do we need this?

Reply:
Because Android Studio's linter highlights this as a security risk and so the ignore tells it not to worry.

Reply:
Done, thanks!